apis/flowcollector/v1beta1/flowcollector_types.go

```diff
@@ -189,72 +189,84 @@ type EBPFMetrics struct {
-	// To filter a single port, set a single port as an integer value. For example, destPorts: 80.
-	// To filter a range of ports, use a "start-end" range in string format. For example, destPorts: "80-100".
+	// `destPorts` optionally defines the destination ports to filter flows by.
+	// To filter a single port, set a single port as an integer value. For example, `destPorts: 80`.
+	// To filter a range of ports, use a "start-end" range in string format. For example, `destPorts: "80-100"`.
 	// To filter two ports, use a "port1,port2" in string format. For example, `ports: "80,100"`.
 	// +optional
 	DestPorts intstr.IntOrString `json:"destPorts,omitempty"`
 
-	// Ports defines the ports to filter flows by. it can be user for either source or destination ports.
-	// To filter a single port, set a single port as an integer value. For example, ports: 80.
-	// To filter a range of ports, use a "start-end" range in string format. For example, ports: "80-100".
+	// `ports` optionally defines the ports to filter flows by. It is used both for source and destination ports.
+	// To filter a single port, set a single port as an integer value. For example, `ports: 80`.
+	// To filter a range of ports, use a "start-end" range in string format. For example, `ports: "80-100"`.
 	// To filter two ports, use a "port1,port2" in string format. For example, `ports: "80,100"`.
 	Ports intstr.IntOrString `json:"ports,omitempty"`
 
-	// PeerIP defines the IP address to filter flows by.
-	// Example: 10.10.10.10
+	// `peerIP` optionally defines the remote IP address to filter flows by.
+	// Example: `10.10.10.10`.
 	// +optional
 	PeerIP string `json:"peerIP,omitempty"`
 
-	// ICMPType defines the ICMP type to filter flows by.
+	// `icmpCode`, for Internet Control Message Protocol (ICMP) traffic, optionally defines the ICMP code to filter flows by.
 	// +optional
-	ICMPType *int `json:"icmpType,omitempty"`
+	ICMPCode *int `json:"icmpCode,omitempty"`
 
-	// ICMPCode defines the ICMP code to filter flows by.
+	// `icmpType`, for ICMP traffic, optionally defines the ICMP type to filter flows by.
 	// +optional
-	ICMPCode *int `json:"icmpCode,omitempty"`
+	ICMPType *int `json:"icmpType,omitempty"`
 
-	// `pktDrops`, to filter flows with packet drops
+	// `pktDrops` optionally filters only flows containing packet drops.
 	// +optional
 	PktDrops *bool `json:"pktDrops,omitempty"`
 }
 
+// `EBPFFlowFilter` defines the desired eBPF agent configuration regarding flow filtering.
+type EBPFFlowFilter struct {
+	// Set `enable` to `true` to enable the eBPF flow filtering feature.
+	Enable *bool `json:"enable,omitempty"`
+
+	// [deprecated (*)] this setting is not used anymore.
+	EBPFFlowFilterRule `json:",inline"`
+
+	// `flowFilterRules` defines a list of ebpf agent flow filtering rules
+	// +kubebuilder:validation:MinItems:=1
+	// +kubebuilder:validation:MaxItems:=16
+	FlowFilterRules []EBPFFlowFilterRule `json:"flowFilterRules,omitempty"`
```

Comment:
```suggestion
	FlowFilterRules []EBPFFlowFilterRule `json:"rules,omitempty"`
```

As this is already under FlowFilter section we should simplify the namming

Reply:
fixed